astronomer/airflow/version_check/update_checks.py

```diff
@@ -250,6 +250,25 @@ def available_update(self):
             AstronomerAvailableVersion.hidden_from_ui.is_(False)
         )
 
+        ac_version = get_ac_version()
+        main_part = ac_version.split(".post")[0]
+        nearest_release = (
+            session.query(AstronomerAvailableVersion)
+            .filter(AstronomerAvailableVersion.version.like(f"{main_part}%"))
+            .filter(AstronomerAvailableVersion.hidden_from_ui.is_(False))
+            .order_by(AstronomerAvailableVersion.date_released.desc())
+            .first()
```

Comment:
Do we need to query the table again ?  We have all the versions already available on L249

Reply:
https://github.com/astronomer/astronomer-airflow-version-check/blob/bf77c2f84bfe46906d8404f21e351f31f1980817/astronomer/airflow/version_check/update_checks.py#L249-L252